WORKSPACE

```diff
@@ -0,0 +1,30 @@
+workspace(name = "com_github_ebitengine_purego")
```

Comment:
What is this file? Is this necesasry on the top directory?

Reply:
I can move it somewhere else but it will have to be at the root of the repo for running gazelle and building any go package

Reply:
It has the same role in a bazel build than the go.mod and go.sum files

Reply:
> I can move it somewhere else but it will have to be at the root of the repo for running gazelle and building any go package

Really? So another question is, do third party Go projects have to have this file for Bazel? I don't think so as I have never seen this file.